src/Options.js

```diff
@@ -487,6 +505,14 @@ export function addOptions(flags, commandOptions) {
     'sourceMaps generated to file or inline with data: URL',
     (to) => { return commandOptions.sourceMaps = to; }
   );
+  flags.option('--source-root <false|string|default>',
```

Comment:
We need to express:
  set the sourceRoot to this value,
  leave the sourceRoot undefined,
  set the sourceRoot to the basepath(output file).
 
What shall we use to signal these choices?